### PR TITLE
Adding tooltips with agent stats to Agent Equipment screen

### LINKED ISSUE #551: CONCEPT Suggestion just for stats look little better
I moved the separation between titles and indicators to the center of the info block and this gave more room for possible problems with long words in translations. Added 2 pixels to the distance between stat names and indicators

statssuggest

statssuggest3

statssuggest4

statssuggest2

### LINKED ISSUE #553: Tooltips with agent stats at equipment screen
It is at OG. So not yet implemented in OA.

It's may be double. e.g. "psi-defence" - one number, "health" - two numbers - current and total (for example "34/41")

### PR DESCRIPTION
**Pull request by Atrosha**
Fixes #553 
This will close my feature request

Fixes #551 
Optional extra (see below)

Hello World! This is my very first PR at GitHub.
I attach some screenshots for demonstration, but it's with old savegame file, so it has some version incompatibility bugs and simple doesn't fixed bugs.

Screenshot from 2019-05-12 08-38-30
Screenshot from 2019-05-12 08-38-39
Screenshot from 2019-05-12 08-40-21

### CODE REVIEW

**Comment by Atrosha** [comment]
This for close my feature request #553

**Comment by Atrosha** [comment]
Can't pass only Travis Lint :-1:

**Comment by makus82** [comment]
Great Work @Atrosha 
i like it, maybe you also will be interested in this)
so i just link it 
https://github.com/OpenApoc/OpenApoc/issues/551

**Comment by Atrosha** [comment]
> #551
Hi, @makus82 , iv'd keep it my mind while making this PR.
IMHO adding tooltips to expand GUI functionality at this section is enough.
Like it is at OA.

I still thinking about your FR but this a realy little work opens the discussion of it reasonable or not.

**Comment by Atrosha** [comment]
The strange server disk error at AppVeyor's Win x64 :-1:

**Comment by makus82** [comment]
> > #551
> 
> Hi, @makus82 , i'd keep it my mind while making this PR.
> IMHO adding tooltips to expand GUI functionality at this section is enough.
> Like it is at OA.
> 
> I still thinking about your FR but this a realy little work opens the discussion of it reasonable or not.

its pure design option so its beyond any reasons))

**Comment by Atrosha** [comment]
I selected this optional design by @makus82
 
Screenshot from 2019-05-12 16-59-07

**Comment by makus82** [comment]
> I selected this optional design by @makus82
> 
> Screenshot from 2019-05-12 16-59-07

dont get this image, what you mean ?

**Comment by Atrosha** [comment]
I was meant this

https://github.com/OpenApoc/OpenApoc/issues/551#issuecomment-491590230

**Comment by Atrosha** [comment]
@makus82 
Макус, здела то, что ты просил. Добавил как дополнительную опцию. Вынес на обсуждение.

**Comment by Atrosha** [comment]
Variant Number 2

Screenshot from 2019-05-12 19-54-18

Variant Number 3

Screenshot from 2019-05-12 20-19-10

**Comment by makus82** [comment]
I thin as separate option its too small twick need to gather some changes)

**Comment by Atrosha** [comment]
**VARIANT 4**

Screenshot from 2019-05-12 20-40-20

**Comment by makus82** [comment]
perfect =]

File: game/ui/general/agentsheet.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include "framework/renderer.h"
 #include "game/state/gamestate.h"
 #include "game/state/rules/battle/damage.h"
+#include <string>
```
Comment (FranciscoDA) [inline game/ui/general/agentsheet.cpp]: It's not necessary to include this here.
If you need to convert numbers to string use some of the functions in `library/strings.h`. You can also use formatting (tinyformat syntax): `format("%d/%d", modified_stats.x, current_stats.x)`

**Comment by FranciscoDA** [comment]
Also make sure you run clang-format on the .cpp/.h files that were modified. Then you can use `git rebase -i HEAD~n` to clean up the history and force-push to your branch

File: game/ui/general/agentsheet.cpp (hunk 1 above)
Comment (Atrosha) [reply game/ui/general/agentsheet.cpp]: I removed `if` expressions. It will be two of stats separated by slash `"Health: 99/99"` or smth like this.

**Comment by FilmBoy84** [comment]
LGTM, i can merge unless JonnyH wants to check?

**Pull request merged**